rabbitmq/assets/configuration/spec.yaml

```diff
@@ -7,131 +7,169 @@ files:
     - template: init_config/http
     - template: init_config/default
   - template: instances
+    multiple_instances_defined: true
     options:
-    - name: rabbitmq_api_url
-      required: true
-      description: |
-        For every instance a 'rabbitmq_api_url' must be provided, pointing to the api
-        url of the RabbitMQ Management Plugin (http://www.rabbitmq.com/management.html).
-      value:
-        type: string
-        example: http://localhost:15672/api/
-    - name: tag_families
-      description: To tag queue "families" based off of regex matching.
-      value:
-        type: boolean
-        example: false
-    - name: collect_node_metrics
-      description: Node metrics are collected by default. Setting this parameter to false skips node metric collection.
-      value:
-        type: boolean
-        example: true
-    - name: nodes
-      description: |
-        Use the `nodes` parameters to specify the nodes you'd like to
-        collect metrics on (up to 100 nodes).
-        If you have less than 100 nodes, you don't have to set this parameter,
-        the metrics are collected on all the nodes by default.
-      value:
-        type: array
-        items:
+    - name: OpenMetrics
+      description: RabbitMQ Prometheus Plugin Example
+      options:
+      - name: prometheus_plugin
+        description: |
+          Settings for talking to the Prometheus plugin of RabbitMQ.
+          Specify this section to trigger metric collection from the Prometheus plugin endpoints.
+        enabled: true
+        options:
+          - name: url
+            enabled: true
+            required: true
+            description: |
+              Base URL for OpenMetrics endpoints.
+            value:
+              type: string
+              example: 'http://localhost:15692'
+          - name: include_aggregated_endpoint
```

Comment:
What happens if this is false?

Reply:
If it's false and no other endpoints are configured, then nothing gets collected. It's mostly there to disable aggregated collection if detailed collection is enabled

Reply:
This is an interesting case. I would actually be tempted to raise an error if no endpoints are specified, since chugging along silently here doesn't make sense.

@fanny-jiang what do you think?

Reply:
https://github.com/DataDog/integrations-core/pull/13793